osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/installation-ws-smartmetering.xsd

```diff
@@ -332,6 +332,11 @@
         type="xsd:hexBinary" minOccurs="0" />
       <xsd:element name="Authentication_key" type="xsd:hexBinary"
         minOccurs="0" />
+      <xsd:element name="Global_encryption_broadcast_key" type="xsd:hexBinary" minOccurs="0" />
```

Comment:
Where do you use Global_encryption_broadcast_key?

Reply:
The unused key is removed.